core/src/main/scala/dx/core/languages/wdl/Runtime.scala

```diff
@@ -122,12 +122,26 @@ case class Runtime(wdlVersion: WdlVersion,
           case other =>
             throw new Exception(s"unexpected ${WdlRuntime.GpuKey} value ${other}")
         }
-        InstanceTypeRequest(minMemoryMB = memoryMB,
-                            minDiskGB = diskGB,
-                            diskType = diskType,
-                            minCpu = cpu,
-                            gpu = gpu,
-                            os = Some(Constants.DefaultExecutionEnvironment))
+        val usedGpuDefault: Boolean = gpu match {
+          case Some(b) => false
+          case None    => true
+        }
+        val usedOtherSysDefaults: Boolean = {
+          runtimeAttrs.runtime.map(_.isDefaultSystemRequirements) match {
+            case Some(b) => b
+            case None    => false
+          }
+        }
```

Comment:
Just one trivial case where we do have the cpu/disks/memory specified but they happen to be equal to the default. Could that happen?

Reply:
It will create an `InstanceTypeRequest` with those requirements and resolve to the instance type that satisfy those requirements. But that will be an explicit override coming form a WF script. This is to resolve to native exec's defaults if there are no overrides